utils/unittest/fixtures.go

```diff
@@ -431,7 +431,10 @@ func BlockWithParentAndSeals(parent *flow.Header, seals []*flow.Header) *flow.Bl
 }
 
 func GenesisFixture() *flow.Block {
-	genesis := flow.Genesis(flow.Emulator)
+	genesis, err := flow.Genesis(flow.Emulator)
```

Comment:
I think it's totally reasonable to make `flow.Genesis` not return an error but rather just panic internally. This code is mostly for testing purposes and we don't expect for genesis to be invalid.